### DISCUSSION TITLE
TimerJobs, intellisense and documentation

### DISCUSSION THREAD

**Discussion by piekenpuil** [pnp/pnpcore#530] [General]
I'm trying to migrate an old TimerJob to PnP.Framework. I have the impression that "job.UseOffice365Authentication" doesn't work anymore. Well, I can live with that, and I understand. I got it working with "job.UseAzureADAppOnlyAuthentication", so I'm happy.

But I'm puzzled: why does intellisense still show "UseOffice365Authentication" if it is not working ? Also no "Deprecated" message is shown. It simply gives back an empty TimerJobRunEventArgs. I'm also unable to find any documentation on this. All I find is the documentation for  PnP-Sites-Core, which I should no longer use. Right ? Am i missing anything ?

**Comment by jansenbe** [comment]
@piekenpuil : This is the wrong place to ask this, belongs more with PnP Framework...but here's some feedback:
- UseOffice365Authentication still can work, but since we use Azure AD auth now it requires the PnP Management Shell app to be configured for the tenant. See https://pnp.github.io/powershell/articles/authentication.html#setting-up-access to learn about setting up the PnP Management Shell app
- PnP Sites Core indeed is not maintained anymore, you can still use it but it will not evolve
